src/banner.ts

```diff
@@ -1,7 +1,8 @@
 import chalk from 'chalk';
 import {readFileSync} from 'fs';
+import path from 'path';
 
-const version = readFileSync('version.txt', 'utf8');
+const version = readFileSync(path.join(__dirname, '../../version.txt'), 'utf8');
```

Comment:
I would really like to get this change in if possible. I'm not terribly familiar with the typescript compiler nor the node runner, but when I run it from my other project, the `version.txt` file isn't found as it's looking for it somewhere else.

Reply:
Interesting... Yeah I'm getting the version running as is, but if you're running within Docker, this could be an issue?

Reply:
I think it was looking in the path of the project that wraps it.